Subject: mac80211: remove max_antenna_gain config

The antenna gain isn't exactly configurable, despite the belief of
some unnamed individual who thinks that the EEPROM might influence
it.

Signed-off-by: Johannes Berg <johannes@sipsolutions.net>
---
 include/net/mac80211.h |    2 --
 net/mac80211/main.c    |    2 --
 2 files changed, 4 deletions(-)

--- everything.orig/include/net/mac80211.h	2008-10-07 20:06:42.000000000 +0200
+++ everything/include/net/mac80211.h	2008-10-07 20:06:43.000000000 +0200
@@ -470,7 +470,6 @@ enum ieee80211_conf_flags {
  * @listen_interval: listen interval in units of beacon interval
  * @flags: configuration flags defined above
  * @power_level: requested transmit power (in dBm)
- * @max_antenna_gain: maximum antenna gain (in dBi)
  * @antenna_sel_tx: transmit antenna selection, 0: default/diversity,
  *	1/2: antenna 0/1
  * @antenna_sel_rx: receive antenna selection, like @antenna_sel_tx
@@ -485,7 +484,6 @@ struct ieee80211_conf {
 	u16 listen_interval;
 	u32 flags;
 	int power_level;
-	int max_antenna_gain;
 	u8 antenna_sel_tx;
 	u8 antenna_sel_rx;
 
--- everything.orig/net/mac80211/main.c	2008-10-07 20:06:43.000000000 +0200
+++ everything/net/mac80211/main.c	2008-10-07 20:06:43.000000000 +0200
@@ -215,8 +215,6 @@ int ieee80211_hw_config(struct ieee80211
 		local->hw.conf.power_level = min(chan->max_power,
 					       local->hw.conf.power_level);
 
-	local->hw.conf.max_antenna_gain = chan->max_antenna_gain;
-
 #ifdef CONFIG_MAC80211_VERBOSE_DEBUG
 	printk(KERN_DEBUG "%s: HW CONFIG: freq=%d\n",
 	       wiphy_name(local->hw.wiphy), chan->center_freq);
